.gitignore

```diff
@@ -27,4 +27,5 @@ sitemap*.xml
 packages/*/typedoc/*
 
 *storybook.log
-storybook-static
\ No newline at end of file
+storybook-static
+.aider*
```

Comment:
what's this?

Reply:
My AI assistant, don't want those files in here